packages/web3/package.json

```diff
@@ -98,4 +98,4 @@
     "node": ">=14.0.0",
     "npm": ">=7.0.0"
   }
-}
+}
```

Comment:
Please revert the end-of-file change to all updated files, maybe it is a editor config issue.

Reply:
It was a mistake. will revert it

Reply:
I have reverted `package.json` in `packages/web3/`